02.calendar/cal.rb

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env ruby
+
+# Dateクラスを使うために宣言
+require 'date'
+
+# optparserクラスを使うために宣言
+require 'optparse'
+
+options = {}
+opt = OptionParser.new
+
+opt.on('-m MONTH'){|m|
+    options[:month] = m.to_i
+}
+
+opt.on('-y YEAR'){|y|
+    options[:year] = y.to_i
+}
+
+opt.parse!(ARGV)
+
+if options[:month] == nil
+    options[:month] = Date.today.month
+end
+
+if options[:year] == nil
+    options[:year] = Date.today.year
+end
+
+if options[:month] < 1 || options[:month] > 12
+    puts "Month can be entered from 1 to 12"
+    abort
```

Comment:
abortのリファレンスマニュアルを再度見てみてください
https://docs.ruby-lang.org/ja/latest/method/Kernel/m/abort.html

> 引数 message を指定すると SystemExit クラスの [Exception#message](https://docs.ruby-lang.org/ja/latest/method/Exception/i/message.html) に message を設定して標準エラー出力に出力します。

と書いている通り、 `abort` 自体にエラーメッセージを引数として指定することで標準エラー出力に出力してくれます。

```suggestion
    abort "Month can be entered from 1 to 12"
```